app/models/standards_import.rb

```diff
@@ -44,6 +44,18 @@ def has_notified_uploader_of_all_conversions?
     source_files.count == source_files.count { |source_file| source_file.courtesy_notification_completed? }
   end
 
+  def files=(files)
+    files.each do |file|
+      imports.build(
+        type: "Imports::Uncategorized",
+        status: :pending,
+        public_document: public_document,
+        file: file
+      )
+    end
```

Comment:
This is messing up a bunch of specs, so I will probably wait to merge this as one of the last pieces to fix.